components/AiTaskInputForm/index.js

```diff
@@ -85,7 +85,7 @@ export default function AiTaskInputForm({ onSubmit, formName, newAiTaskData }) {
           ref={original_task_descriptionInputRef}
           placeholder="e.g. plan a trip to Mallorca"
         />
-        <ButtonsContainer>
+        <ButtonsContainer variant="fixed">
           <NavigationLinkWrapper>
             <Link href={`/`} passHref legacyBehavior aria-hidden="true">
```

Comment:
Your Link / Button combinations are somehow corrupting the Link functionality.
1. Please create `StyledLink`s so that they have a correct and clean behaviour (including mouseOver pointer etc.) and you can get rid of these odd `Link` props like `passHref`, `legacyBehavior` and `aria-hidden` (why do you set this?) and remove your buttons inside `Link` components. 
2. Please create Icon components of your svgs, makes the code easier to read and maintain.